### PR TITLE
Image card

### PR DESCRIPTION
**Pull request by robert-little**
Added a bunch of stuff, both functionality and styling.

### CODE REVIEW

File: App/Components/Styles/ImageCardStyle.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  textContents: {
+    flex: 80,
+    overflow: 'hidden'
+  },
+  name: {
+    color: 'white',
+    paddingLeft: 10,
+    fontSize: 30
+  },
+  catPicture: {
```
Comment (mctaggaj) [inline App/Components/Styles/ImageCardStyle.js]: I take it this is from your previous project?

File: WhatIsInMyPocket/Screens/PresentationScreen.js [hunk 2]
```diff
@@ -116,20 +118,22 @@ export default class PresentationScreen extends React.Component {
       'Content-Type': 'multipart/form-data',
     })
 
-    var data = new FormData();
-    data.append('authToken', 'wb7J1G536cI2yvm');
-    data.append('id', query.item_id);
-    db.get('/polka.php', data)
+    var url = '/polka.php?token=' + query.key
+
+    console.log(url)
```
Comment (mctaggaj) [inline WhatIsInMyPocket/Screens/PresentationScreen.js]: Console log

File: WhatIsInMyPocket/Screens/PresentationScreen.js [hunk 3]
```diff
@@ -116,20 +118,22 @@ export default class PresentationScreen extends React.Component {
...
+
+    db.get(url)
       .then((response) => {
+        console.log(response.data)
```
Comment (mctaggaj) [inline WhatIsInMyPocket/Screens/PresentationScreen.js]: Console.log

File: WhatIsInMyPocket/Screens/PresentationScreen.js [hunk 4]
```diff
@@ -116,20 +118,22 @@ export default class PresentationScreen extends React.Component {
...
-          image: query.photo,
+          image: query.image,
           state: response.data.status,
           amount: response.data.amount,
           coins: response.data.coins,
           item_id: query.item_id,
           key: query.key
         };
-        if (response.data.status !== 1) {
+        if (response.data.status === 1) {
```
Comment (mctaggaj) [inline WhatIsInMyPocket/Screens/PresentationScreen.js:120]: '1' means that the result is ready no?

File: WhatIsInMyPocket/Screens/PresentationScreen.js [hunk 5]
```diff
@@ -180,73 +184,68 @@ export default class PresentationScreen extends React.Component {
       queries: queries,
       avatarSource: null
     });
+
+    console.log("in the upload")
```
Comment (mctaggaj) [inline WhatIsInMyPocket/Screens/PresentationScreen.js]: console log

File: WhatIsInMyPocket/Screens/PresentationScreen.js [hunk 6]
```diff
@@ -180,73 +184,68 @@ export default class PresentationScreen extends React.Component {
...
+
     db.post('/urload.php', data)
-    .then(() => {
+    .then(function (response) {
+      console.log("here");
```
Comment (mctaggaj) [inline WhatIsInMyPocket/Screens/PresentationScreen.js]: console log

File: WhatIsInMyPocket/Screens/PresentationScreen.js [hunk 7]
```diff
@@ -180,73 +184,68 @@ export default class PresentationScreen extends React.Component {
...
-    // .then((responseData) => {
-    //     AlertIOS.alert(
-    //         "POST Response",
-    //         "Upload Test -> " + JSON.stringify(responseData)
-    //       )
-    // })
-    // .done();
-
+  cardClicked(event) {
+    console.log(event)
```
Comment (mctaggaj) [inline WhatIsInMyPocket/Screens/PresentationScreen.js]: Console log

File: WhatIsInMyPocket/Screens/PresentationScreen.js [hunk 8]
```diff
@@ -180,73 +184,68 @@ export default class PresentationScreen extends React.Component {
...
     var rows = [];
     for (var i = 0 ; i < this.state.queries.length ; i ++) {
-      rows.push(<Text key="{this.state.queries[i].key}">id: {this.state.queries[i].item_id} amount:{this.state.queries[i].amount}</Text>);
+      rows.unshift(<ImageCard queryData={this.state.queries[i]} onClick={this.cardClicked.bind(this)} key={this.state.queries[i].key}/>);
     }
 
-    return (
-      <View style={styles.container}>
+    var mainPic = this.state.avatarSource
+    console.log(mainPic)
```
Comment (mctaggaj) [inline WhatIsInMyPocket/Screens/PresentationScreen.js]: log

**Review by mctaggaj — CHANGES_REQUESTED** [review]

File: App/Components/Styles/ImageCardStyle.js (hunk 1 above)
Comment (robert-little) [reply App/Components/Styles/ImageCardStyle.js]: lol yup good catch

File: WhatIsInMyPocket/Screens/PresentationScreen.js (hunk 4 above)
Comment (robert-little) [reply WhatIsInMyPocket/Screens/PresentationScreen.js:120]: No 0 is ready 1 is still working

**Pull request merged**